pkg/image/qemu_test.go

```diff
@@ -301,7 +301,7 @@ var _ = Describe("Create blank image", func() {
 		size := convertQuantityToQemuSize(quantity)
 		replaceExecFunction(mockExecFunction("", "", nil, "create", "-f", "raw", "image", size), func() {
 			err = CreateBlankImage("image", quantity, false)
-			Expect(err).NotTo(HaveOccurred())
+			Expect(err.Error()).To(ContainSubstring("no such file or directory"))
```

Comment:
hmm are we fine with this error happening? shouldn't we pass a tempdir to CreateBlankImage and keep test the same?

Reply:
Well that error is happening because the file we pass in is not real, and since we were ignoring the error before...

Reply:
I think it may be worth to create a temp file and avoid the err, I think we do this in other qemu tests already

Reply:
Let me know if you want to do that